tests/torch/quantization/test_functions.py

```diff
@@ -92,12 +93,93 @@ def idfn(val):
 
 @pytest.fixture
 def _seed():
-    np.random.seed(0)
-
+    np.random.seed(42)
```

Comment:
Just out of curiosity, why did you change this?

Reply:
Glad you ask! It's just for stability :) (as you know, seed 42 is the most stable)